2021-04-05-announcements.md

```diff
@@ -0,0 +1,39 @@
+# CFDE Announcements
+Hello CFDE friends! Our list of events continues to grow. Check our list of Demos and Events and scroll down for Working Group updates.
+
+## CFDE Program Meeting
+A major CFDE Program meeting is scheduled for **May 10-11, 2021** with presentations and activities taking place from 11am EST until 4pm EST (8am-1pm PST) on both days.  All CFDE participants are encouraged to attend two days of workshops and talks on a range of topics. A calendar invitation has already gone out. If you have questions or concerns about this important meeting, contact us at support@cfde.atlassian.net.
+
+## Cross-Pollination Events
+The next event will be on **Tuesday, April 6, at 11am PST/2pm EST**.Regular Cross-Pollination events take place on the first Tuesday of every month. Please check out the [Cross-Pollination Schedule](https://docs.google.com/spreadsheets/d/1hQAeOLkivUZZnwZ_KxfGw3neezMaWbrPk9nnFiKfQGA/edit?usp=sharing) and let us know if you would like to present! When requesting a slot please include your preferred date and the length of your talk. Currently we have speakers scheduled out through June, but you can book a slot from July onwards. 
+
+## CFDE Portal Demo
+A 1-hour pop-up demo of newly released CFDE Data Portal that allows you to search across data from HuBMAP, Kids First Data Resource Center, Metabolomics, LINCS, exRNA, GTEx, and HMP from a single website is planned for **Wednesday, April 14, at 11am PDT / 2pm EDT**. Come check out what the portal has to offer and where we'll be taking it next. You can sign up for the event by clicking on the [event calendar link](https://www.nih-cfde.org/events/cfde-portal-demo/?pk_campaign=anc).
+
+## CFDE Trainings and Workshops
+Our Workshop series continues in April with lessons on AWS, Conda, and more. On **Wednesday, April 7, at 10am PST/1pm EST** we will host another 2 hour session on [Intro to Conda](https://www.nih-cfde.org/events/intro-to-conda/). More lessons and sessions are being added and can be see on the [CFDE Events Calendar](https://www.nih-cfde.org/events/intro-to-conda/?pk_campaign=anc). We are committed to developing training workshops that will equip the CFDE with resources to enhance their research and collaboration efforts! Our Common Fund Training & Outreach Team would like your input on what lessons we can design to best achieve these goals in collaboration with your teams and users. Please take a moment to fill out our Google form and let us know where you think we can maximize our impact: [CFDE Training and Workshop Survey](https://forms.gle/48hGnrEfiHXJZtUs7).
```

Comment:
the campaign link should be for the conda event, the [CFDE Events Calendar] can be just to the calendar itself